### PR TITLE
Classification demo: fixed zero division exception, added feature to repeat testing.

### PR DESCRIPTION
**Pull request by ivikhrev**
In a few cases might occur zero division exception ( for example - when you try to open GUI on remote machine, imshow took long time to open and `targetFPS` parameter got 0). Also added feature that allows repeating testing.

### CODE REVIEW

File: demos/classification_demo/cpp/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The list of models supported by the demo is in `<omz_dir>/demos/classification_d
 This file can be used as a parameter for [Model Downloader](../../../tools/downloader/README.md) and Converter to download and, if necessary, convert models to OpenVINO Inference Engine format (\*.xml + \*.bin).
 
 An example of using the Model Downloader:
-
+-
```
Comment (Wovchena) [inline demos/classification_demo/cpp/README.md:23]:
```suggestion

```

Comment (Wovchena) [reply demos/classification_demo/cpp/README.md]: This should be empty line: https://github.com/openvinotoolkit/open_model_zoo/commit/c7d1a8b242a99793e1ade36fa219d3f4d780f6d2. Strangely github interpreted the suggestion as line removal.

File: demos/classification_demo/cpp/README.md [hunk 2]
```diff
@@ -20,7 +20,6 @@ The list of models supported by the demo is in `<omz_dir>/demos/classification_d
 This file can be used as a parameter for [Model Downloader](../../../tools/downloader/README.md) and Converter to download and, if necessary, convert models to OpenVINO Inference Engine format (\*.xml + \*.bin).
 
 An example of using the Model Downloader:
```
Comment (ivikhrev) [inline demos/classification_demo/cpp/README.md:22]:
```suggestion
An example of using the Model Downloader:

```

File: demos/classification_demo/cpp/README.md (hunk 1 above)
Comment (ivikhrev) [reply demos/classification_demo/cpp/README.md]: Restored empty line.

**Review by Wovchena — APPROVED** [review]

**Review by vladimir-dudnik — APPROVED** [review]
do we need this fix in release branch?

**Comment by Wovchena** [comment]
We lived with this bug fine for some time, so no. But is there a process prescribing how we decide what we need and what we don't in first place?

**Comment by vladimir-dudnik** [comment]
After code freeze only bug fixes and doc improvements are allowed (no new features are expected, so no new model, no new demo should appear). If this is a minor issue, which does not prevent demo from running then might be postponed until next release.

**Comment by Wovchena** [comment]
Jenkins please retry a build

**Pull request merged**